test/request-response-samples/RequestHeaders__propogate_basic_security backward compatible check/request.xml

```diff
@@ -1 +1 @@
-<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ><soap:Header><wsse:Security xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd" xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"><wsu:Timestamp wsu:Id="Timestamp-2014-10-12T01:02:03Z"><wsu:Created>2014-10-12T01:02:03Z</wsu:Created><wsu:Expires>2014-10-12T01:12:03Z</wsu:Expires></wsu:Timestamp><wsse:UsernameToken xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd" wsu:Id="SecurityToken-2014-10-12T01:02:03Z"><wsse:Username>basicuser</wsse:Username><wsse:Password Type="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-username-token-profile-1.0#PasswordText">basicpass</wsse:Password><wsu:Created>2014-10-12T01:02:03Z</wsu:Created></wsse:UsernameToken></wsse:Security></soap:Header><soap:Body><Request xmlns="http://www.example.com/v1"></Request></soap:Body></soap:Envelope>
```

Comment:
It looks like a newline was added automatically; I'll remove these files from the new PR to make it more clear what's changing.